### PR TITLE
[UX2.0] Adds Transport GPS Profile Parcel

### PR DESCRIPTION
**Pull request by seconroy**
### Description

Adds the Transport GPS Profile Parcel resources and data source

### Types of Changes

* [x] New feature (non-breaking change which adds functionaality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Build/CI change
* [ ] Code quality improvement/refactoring/documentation (no functional changes)

### Checklist

* [x] My code follows the code style of this project
* [x] I have added tests to cover my changes
* [x] All new and existing tests pass locally

### CODE REVIEW

File: gen/definitions/profile_parcels/transport_gps.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_transport_feature_profile.test.id
+  - model_name: enable
+    example: false
+  - model_name: mode
+    example: "ms-based"
+  - model_name: nmea
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/transport_gps.yaml:18]: should we add tf_name: nmea_enable to differentiate from other options below?

File: gen/definitions/profile_parcels/transport_gps.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    example: false
+  - model_name: sourceAddress
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/transport_gps.yaml:20]: I would put "tf_name: nmea_source_address" as this option is used only for nmea and if nmea is enabled

File: gen/definitions/profile_parcels/transport_gps.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    example: 1.2.3.4
+  - model_name: destinationAddress
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/transport_gps.yaml:22]: I would put "tf_name: nmea_destination_address" as this option is used only for nmea and if nmea is enabled

File: gen/definitions/profile_parcels/transport_gps.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    example: 2.3.4.5
+  - model_name: destinationPort
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/transport_gps.yaml:24]: I would put "tf_name: nmea_destination_port" as this option is used only for nmea and if nmea is enabled

File: gen/definitions/profile_parcels/transport_gps.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+test_tags: [SDWAN_2012]
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_transport_feature_profile.test.id
+  - model_name: enable
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/transport_gps.yaml:14]: If we modify "nmea" to "nmea_enable" then here I would put here "gps_enable"

File: gen/definitions/profile_parcels/transport_gps.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    example: false
+  - model_name: mode
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/transport_gps.yaml:16]: "tf_name: gps_mode"

**Review by danischm — APPROVED** [review]

**Pull request merged**